.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 - pip install --upgrade pip
 - pip --version
 - pip install -r pip-freeze.txt --upgrade
+- pip install setuptools==33.1.1  # Recent version breaks packaging.requirements, this is a temp fix: https://github.com/pypa/setuptools/issues/942
```

Comment:
it was failing last week when I wrote this, I'll remove it again as hopefully it's been fixed now. Thanks for spotting.